packages/client/tests/functional/_utils/getTestSuiteInfo.ts

```diff
@@ -3,20 +3,27 @@ import path from 'path'
 import { matrix } from '../../../../../helpers/blaze/matrix'
 import { merge } from '../../../../../helpers/blaze/merge'
 import { MatrixTestHelper } from './defineMatrix'
-import { isDriverAdapterProviderFlavor, ProviderFlavors, Providers } from './providers'
+import { isDriverAdapterProviderFlavor, ProviderFlavors, Providers, RelationModes } from './providers'
 import type { TestSuiteMeta } from './setupTestSuiteMatrix'
 import { ClientMeta, TestCliMeta } from './types'
 
 export type TestSuiteMatrix = { [K in string]: any }[][]
 export type NamedTestSuiteConfig = {
   parametersString: string
-  matrixOptions: Record<string, string> & { provider: Providers; providerFlavor?: ProviderFlavors }
+  matrixOptions: Record<string, string> & {
+    provider: Providers
+    providerFlavor?: ProviderFlavors
+    relationMode?: RelationModes
```

Comment:
I formalized what relation mode is withing the test setup itself, so now an understood concept.